packages/message/src/StepFunctions.ts

```diff
@@ -8,14 +8,35 @@
  * const StepFunctions = require('@cumulus/message/StepFunctions');
  */
 
+import { EventBridgeEvent } from 'aws-lambda';
 import { JSONPath } from 'jsonpath-plus';
+import get from 'lodash/get';
+import set from 'lodash/set';
+
+import { getExecutionHistory } from '@cumulus/aws-client/StepFunctions';
+import { getStepExitedEvent, getTaskExitedEventOutput } from '@cumulus/common/execution-history';
+import { Message } from '@cumulus/types';
 import * as s3Utils from '@cumulus/aws-client/S3';
 import Logger from '@cumulus/logger';
-import { Message } from '@cumulus/types';
+
+import { getMessageExecutionArn } from './Executions';
 
 const log = new Logger({
   sender: '@cumulus/message/StepFunctions',
 });
+type ExecutionStatusToWorkflowStatus = ('ABORTED' | 'FAILED' | 'RUNNING' | 'SUCCEEDED' | 'FAILED');
+
+const executionStatusToWorkflowStatus = (executionStatus: ExecutionStatusToWorkflowStatus) => {
+  const statusMap = {
```

Comment:
Would it be superfluous to add a type here of `ExecutionStatusToWorkflowStatusMap` which is defined as 

```
{ [ExecutionStatusToWorkflowStatus]: WorkflowStatus }
```

where `WorkflowStatus` comes from `@cumulus/types/message`. I think this would ensure that we only map to valid workflow status values

Reply:
An excellent suggestion 👍🏻

Reply:
33c398f8b